### PR TITLE
added hashtags,edited  upload page

### PR DESCRIPTION
**Pull request by hodanourbakhsh**
This PR addresses the user story #49 and #50. The upload page layout has been completed by editing some details for the UI. The hashtags are also added into the description for the posts.

### CODE REVIEW

**Comment by po-trottier** [comment]
Any Updates on This ?

**Comment by hodanourbakhsh** [comment]
To be honest I couldn’t work on it yet, but I’ll try to finish it by
tomorrow night.

**Comment by hodanourbakhsh** [comment]
Required changes have been applied. Please take a look. @po-trottier

File: frontend/src/components/Details/Sidebar.vue [hunk 1]
```diff
@@ -21,7 +21,8 @@
       </span>
     </v-row>
     <div class="mb-4">
-      <p class="mb-2">
+      <p
+        class="mb-2">
```
Comment (po-trottier) [inline frontend/src/components/Details/Sidebar.vue]: Put the class on the same line

File: frontend/src/components/Feed/PostActions.vue [hunk 2]
```diff
@@ -22,7 +22,8 @@
       class="px-3"
       style="width: 100%;">
       <v-row no-gutters>
-        <p class="mb-2">
+        <p
+          class="mb-2">
```
Comment (po-trottier) [inline frontend/src/components/Feed/PostActions.vue]: Put the class on the same line

File: frontend/src/components/Details/Sidebar.vue [hunk 3]
```diff
@@ -86,6 +86,9 @@ export default {
     ...mapGetters('details', ['getPost']),
     ...mapGetters('auth', ['user']),
 
+    descriptionWithTags() {
+      return this.getPost.description.replace(/(^|\s)(#[a-zA-Z\d-]+)/ig, "$1<a href='#$2'>$2</a>");
```
[the remark this answers is not in the document]
Comment (po-trottier) [reply frontend/src/components/Details/Sidebar.vue]: You didn't change this

File: frontend/src/components/Feed/PostActions.vue [hunk 4]
```diff
@@ -59,6 +59,9 @@ export default {
 
   computed: {
     ...mapGetters('auth', ['user']),
+    descriptionWithTags() {
+      return this.content.description.replace(/(^|\s)(#[a-zA-Z\d-]+)/ig, "$1<a href='#$2'>$2</a>");
```
[the remark this answers is not in the document]
Comment (po-trottier) [reply frontend/src/components/Feed/PostActions.vue]: You didn't change this

**Comment by hodanourbakhsh** [comment]
Oki!

**Comment by hodanourbakhsh** [comment]
I have made the changes. Please take a look. @po-trottier

**Pull request merged**